### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by semenetsoleksandr**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = crlf
```
Comment (sergiiluchko) [inline .editorconfig]: @semenetsoleksandr , could you please use lf for end of line

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1 @@
-.DS_Store
-
+.DS_Stores
```
Comment (sergiiluchko) [inline .gitignore]: @semenetsoleksandr , it was correct to use `.DS_Store` for MacOS, and please add `Thumbs.db` this is equivalent for Windows. Add popular editors’ cache directories. For WebStorm - `.idea`, VSCode.

**Review by sergiiluchko — CHANGES_REQUESTED** [review]

File: .editorconfig (hunk 1 above)
Comment (semenetsoleksandr) [reply .editorconfig]: set lf for end of line

File: .gitignore (hunk 2 above)
Comment (semenetsoleksandr) [reply .gitignore]: update .gitignore file

**Review by sergiiluchko — APPROVED** [review]

**Pull request merged**